doc/Language/functions.rakudoc

```diff
@@ -1018,6 +1018,28 @@ dispatch, or it could be the inner routine from a C<wrap>.
 Fortunately, we have a series of re-dispatching tools that help us to make
 it easy.
 
+=head2 The Next Tools
+
+=begin table
+
+                    | Returns | Parameters | Referee
```

Comment:
Referee as in reciprocal of referer?

Reply:
Yes; thanks to your comment, I wrote some text to explain the table.  Hopefully it helps.